scripts/compare_records_in_two_indices.py

```diff
@@ -0,0 +1,83 @@
+"""
+compare two versions of same type records stored in the two different indices
+"""
+import click
+import constants
+import requests
+from typing import Dict
+
+
+@click.command()
+@click.option(
+    '--es_host',
+    default="http://wp-np3-e2:9200",
+    help='Specify the Elastic Search host, default to be wp-np3-e2'
+)
+@click.option(
+    '--es_index_1',
+    help='Specify the first index, mandatory field'
+)
+@click.option(
+    '--es_index_2',
+    help='Specify the second index, mandatory field'
+)
+@click.option(
+    '--es_type',
+    help='Specify the type of data to be comapred, mandatory field'
+)
+def main(es_host, es_index_1, es_index_2, es_type):
+    """
+    The main function
+    :param es_host:
```

Comment:
It would be great to add documentation about parameters here.

Reply:
done